lib/modules/orangered.js

```diff
@@ -171,9 +166,32 @@ const setupFaviconBadge = _.once(() => {
 	return favicon;
 });
 
-function setupFloatingButtons() {
-	if (!module.options.showFloatingEnvelope.value) return;
-	if ($floatingInboxCount) return;
+const orangeredElements = () => ({
+	...nativeButtons(),
+	...floatingButtons(),
+	$modmailButton: $('#modmail'),
+});
+
+const $orangeredElements = () => {
+	const dict = orangeredElements();
+	const elements = Object.keys(dict).map(key => dict[key]);
```

Comment:
Nah Babel transforms it https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/1ef2c155b88c2dbbdd4a36d2b97ce1947df239ba/lib/core/profiling.js#L99-L101